guides/source/routing.md

```diff
@@ -1386,15 +1462,31 @@ URI               | /users/:id/edit(.:format)
 Controller#Action | users#edit
 ```
 
-You can search through your routes with the grep option: -g. This outputs any routes that partially match the URL helper method name, the HTTP verb, or the URL path.
+### Routes in Rails Console
```

Comment:
The previous section mentioned `bin/rails routes --expanded`.
The next section mentions `bin/rails routes -g new_comment`.
After that `bin/rails routes --unused`.
Maybe move this console section after all `bin/rails routes` sections?

Reply:
makes sense